src/components/reward-claimed-toast/RewardClaimedToast.module.css

```diff
@@ -0,0 +1,22 @@
+.rewardClaimedToast {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.rewardClaimedToastIcon {
+  margin-inline-end: 8px;
```

Comment:
Interesting, i've never seen this style prop before

Reply:
in the future, if we start to support RTL, this will make the margin be on the left instead of the right!